### DISCUSSION TITLE
Apply to add TDengine to the Superset data source list

### DISCUSSION THREAD

**Discussion by DuanKuanJun** [apache/superset#31619] [Ideas]
Hello, I am a development engineer for the time series database TDengine. We currently support the development of TDengine as a SuperSet data source and hope to add our connection driver files to the SuperSet installation package. A few days ago, your team also contacted us and hoped that TDengine could be added to SuperSet.

https://docs.tdengine.com/third-party/bi/superset/
![image](https://github.com/user-attachments/assets/0b4a21cc-b637-43c3-b1d1-f12c8a0fdbd0)

We have completed the merge branch on the superset repository, but we do not have permission to push a branch for review. Can I add my GitHub account permission to submit a PR to you and request a review,
My GitHub account:
 https://github.com/DuanKuanJun

Thank you.

**Comment by hainenber** [comment]
You should fork the `apache/superset` repo, push your change onto the remote forked and open a PR to merge into `apache/superset` master branch.
